timelapse.sh

```diff
@@ -58,5 +58,5 @@ ln -sf $yesterday_dir/timelapse.mp4 $PIC_DIR/latest.mp4
 
 echo "Timelapse complete. Waiting 100 seconds before preloading it into cloudflare cache"
 sleep 100
-curl "https://isitfoggy.today/photos/latest.mp4?$(( ( ( $(date +%s) / 3600) -11)/24 ))" -o /dev/null
+curl "https://${HOST_TO_PROBE}/photos/latest.mp4?$(( ( ( $(date +%s) / 3600) -11)/24 ))" -o /dev/null
```

Comment:
so HOST_TO_PROBE is unclearly documented but it's meant to be the DNS entry for the origin IP
To go through the CDN, you want to use DOMAIN

I should definitely pick better names for these variables